CONTRIBUTING.md

```diff
@@ -107,3 +107,54 @@ MyButton : Add unit test
 
 Files changed:
 - MyButtonTest.py
+
+
+## Code reviews and labels ##
+
+### Label guidelines ###
+For labels, our primary concern is to clearly and consistently distinguish between Issue labels and Pull Request (PR) labels. 
+
+For an up-to-date list of Gaffer's GitHub labels, see the project's [Labels](https://github.com/GafferHQ/gaffer/labels) page.
+
+We have the following general specs for labels:
+- Unfortunately, there is currently no way to maintain separate lists of labels based on their intended use. Therefore, PR labels are prefixed with `pr-`.
+- PR labels that communicate that the PR needs immediate attention have high-intensity colors.
+- PR labels that communicate that the PR should not be merged yet have medium-intensity colors.
+- Issue labels have no prefix, and have low-intensity colors.
+
+#### Issue labels ####
```

Comment:
Should Issue _Label Guidelines_ and _Issue Labels_ be pulled up into the _Reporting Bugs_ section?

Reply:
That sounds good to me. I don't think leading off this section with a discussion of label specs really helps a contributor - it's the review/pr process that they need to know about, and is what we should lead off with. We can document the rationale for colouring etc separately if we must, but what a contributor needs to know is what we mean by pr-revision, and when they should remove it.

Reply:
> Should Issue Label Guidelines and Issue Labels be pulled up into the Reporting Bugs section?

https://github.com/GafferHQ/gaffer/pull/3266/commits/2fbddd227f035e24e9f067628b74d82ec0c8e692#diff-6a3371457528722a734f3c51d9238c13R15-R22

> it's the review/pr process that they need to know about, and is what we should lead off with

https://github.com/GafferHQ/gaffer/pull/3266/commits/2fbddd227f035e24e9f067628b74d82ec0c8e692#diff-6a3371457528722a734f3c51d9238c13R120-R121